benches/event.rs

```diff
@@ -13,7 +13,7 @@ fn benchmark_event_iterate(c: &mut Criterion) {
     group.bench_function("single-level", |b| {
         b.iter_batched_ref(
             || {
-                let mut log = Event::new_empty_log().into_log();
+                let mut log = LogEvent::new();
```

Comment:
Oops, I guess that should have been `LogEvent::default`

Reply:
Addressed in 34ba5665e.